Tests/AES-GCM/AESV8ProgramTests.lean

```diff
@@ -351,7 +351,7 @@ def buf_res_128 : List (BitVec 8) :=
 
 -- len = 0
 def final_state0 : ArmState :=
-  aes_hw_ctr32_encrypt_blocks_test 30 0 in_block rounds key_schedule ivec
+  aes_hw_ctr32_encrypt_blocks_test 28 0 in_block rounds key_schedule ivec
```

Comment:
`28` appears to be the number of instructions in the program, right? Can it be replaced with `program.length`?

Reply:
Good question. This function has a loop and branching controls so it might be hard to calculate the steps through `program.length`.